sysdump/sysdump.go

```diff
@@ -1098,6 +1100,7 @@ func (c *Collector) submitHubbleFlowsTasks(_ context.Context, pods []*corev1.Pod
 			if err := c.WriteBytes(fmt.Sprintf(hubbleObserveFileName, p.Name), e.Bytes()); err != nil {
 				return fmt.Errorf("failed to write hubble stderr for %q in namespace %q: %w", p.Name, p.Namespace, err)
 			}
+			c.logDebug("Finished collecting hubble flows from %s/%s", p.Namespace, p.Name)
```

Comment:
added a debug log here. initially i was looking at this debug log: https://github.com/cilium/cilium-cli/blob/475f31829e32d4554660044709bf21e872d95ac0/sysdump/sysdump.go#L943 but realized this is incorrect, it doesn't wait until the task completes. couldn't figure out how to fix this.